guides/common/modules/con_virtual-machine-subscriptions-overview.adoc

```diff
@@ -0,0 +1,33 @@
+[id="virtual-machine-subscriptions-overview_{context}"]
+= Virtual machine subscriptions overview
+
+Virtual machines (VMs) require host-based subscriptions instead of physical subscriptions.
+Many host-based subscriptions include entitlements for unlimited VMs.
+
+You can configure and deploy virt-who on your hypervisors.
+virt-who queries the virtualization platform and reports hypervisor and VM information to {Project}.
+You can view your subscription usage by using the Subscriptions service on the {RHCloud}.
+
+You can configure host-based subscriptions for {EL} VMs on the following virtualization platforms:
+
+* VMware vSphere
+* {EL} Virtualization (KVM)
+* {OpenStack}
+* {oVirt}
+* Microsoft Hyper-V
+* Nutanix AHV
+* {KubeVirt}
+ifdef::satellite[]
++
+[IMPORTANT]
+====
+The {KubeVirt} hypervisor is a Technology Preview feature only.
```

Comment:
This text is (almost?) identical to `guides/common/modules/con_provisioning-virtual-machines-on-kubevirt.adoc`. Can you please make it a snippet?